docs/lab/02/02.md

```diff
@@ -755,9 +776,9 @@ The maximum current that GPIO pins output depends on the MCU. To be sure that th
 :::
 
 2. Write a Rust program that set on HIGH the LED connected to GPIO pin 0 (GP0). (**2p**)
-   1. use bare metal
+   1. use the `embassy-rs` framework
```

Comment:
I would put all the exercises with embassy at the beginning, and the PAC and bare metal after.

Reply:
done